bin/prettier.js

```diff
@@ -122,14 +98,30 @@ function getIntOption(optionName) {
   process.exit(1);
 }
 
+function getTrailingComma() {
+  let trailingComma;
+  switch(argv["trailing-comma"]) {
+    case undefined:
+    case "none":
+      return "none";
+    case "":
+    case "es5":
+      return "es5";
+    case "all":
+      return "all";
+    default:
+      throw new Error("Invalid option for --trailing-comma");
+  }
```

Comment:
Could you add support for the previous options with a warning saying that they are deprecated. We should try avoid doing breaking changes if humanly possible, they are the best way to kill a project by churning people that build an ecosystem around.

A better way to go around is to make the old options still work and deprecated, and at some point in the future we'll remove them when the majority of users has upgraded due to the exponential adoption curve.